### PR TITLE
WRN-4405: Migrate MediaPlayer, SliderButton and Spinner unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for MediaPlayer, SliderButton and Spinner components.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-4405

### CODE REVIEW

File: MediaPlayer/tests/MediaPlayer-specs.js [hunk 1]
```diff
@@ -23,104 +17,100 @@ describe('MediaPlayer', () => {
...
 				</MediaPlayer>
 			);
+			const expected = screen.getByTestId('media-player');
 
-			const expected = getSourceNode(subject);
-
-			subject.setProps({
-				source: ['abc.mp3']
-			});
+			rerender(
```
Comment (alexandrumorariu) [inline MediaPlayer/tests/MediaPlayer-specs.js:30]: Please write all the rerenders in a single line.

File: Spinner/tests/Spinner-specs.js [hunk 2]
```diff
@@ -1,168 +1,136 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
 
 import Spinner from '../Spinner';
```
Comment (alexandrumorariu) [inline Spinner/tests/Spinner-specs.js:5]: Delete this empty line.

**Review by alexandrumorariu — APPROVED** [review]
LGTM

File: SliderButton/tests/SliderButton-specs.js [hunk 3]
```diff
@@ -1,113 +1,101 @@
...
+		const sliderButton = screen.getByRole('slider');
 
-		sliderButton.find('SliderButton').simulate('change', evt);
+		activate(sliderButton);
+		rightKeyDown(sliderButton);
 
 		const expected = 0;
 		const actual = handleChange.mock.calls.length;
+
 		expect(actual).toBe(expected);
```
Comment (adrian-cocoara-lgp) [inline SliderButton/tests/SliderButton-specs.js:73]:
```suggestion
		test('should not emit onChange event when disabled', () => {
		const handleChange = jest.fn();
		render(<SliderButton disabled onChange={handleChange}>{children}</SliderButton>);
		const sliderButton = screen.getByRole('slider');

		activate(sliderButton);
		rightKeyDown(sliderButton);

		expect(handleChange).not.toHaveBeenCalled();
	});
```

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**